.github/workflows/check-terraform-module-version.yaml

```diff
@@ -8,8 +8,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Check for editted files for terraform modules
-        uses: ministryofjustice/cloud-platform-environments/cmd/check-terraform-modules-are-latest@add-gh-action-to-pin-terraform-modules-to-latest 
+      - name: Check editted files for terraform modules
```

Comment:
Small change: `Check edited files for terraform modules`

Reply:
Thanks!